tests/comfy_cli/command/test_distribution.py

```
@@ -1457,3 +1457,37 @@ def test_create_execute_does_not_read_a_renamed_pack_as_a_dropped_one(monkeypatc
+def test_from_snapshot_refuses_json_that_is_not_an_object(tmp_path, payload):
+    """`json.loads` accepts all of these, and the wrapper would raise AttributeError
+    instead of the error envelope a caller can read."""
+    f = tmp_path / "export.json"
+    f.write_text(payload, encoding="utf-8")
+    with pytest.raises(typer.Exit):
+        distribution.from_snapshot_cmd(from_=str(f), name="demo")
+
+
+def test_the_from_path_is_not_shipped_as_telemetry():
+    """`--from` is a local path naming the user's home directory and install layout."""
+    from comfy_cli.tracking import filter_command_kwargs
```

Pylint (4.0.6)

[error] Unable to import 'comfy_cli.tracking'

(E0401)